tests/network/vmi_istio.go

```diff
@@ -179,7 +179,7 @@ var istioTests = func(vmType VmType) {
 				libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(virtClient, migration)
 			})
 			It("All containers should complete in source virt-launcher pod after migration", func() {
-				const containerCompletionWaitTime = 60
+				const containerCompletionWaitTime = 240
```

Comment:
Does the new passt configuration somehow affects the integration with Istio?
I don't remember seeing this failure in the past few weeks.